tech/languages/csharp/dotnet-installation.md

```diff
@@ -4,18 +4,29 @@ subsection: csharp
 order: 3
 ---
 
-# .NET SDK installation
+# .NET Core installation
 
-.NET Core is still work in progress, and below download locations contain packages that are not entirely _free._ Please be patient, we should have reasonable packages within a few months.
+## Fedora .NET SIG repository
 
-Choose based on your Fedora and SDK version:
+.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as containing prebuilt binaries or bundled dependencies.) Please be patient, we should have reasonable packages within a few months.
```

Comment:
For this to be clear, I'd mention that 'as of now' it's available only in copr repository below.

The other (in Fedora) sources are therefore outdated, right? Maybe it's something worth mentioning in case someone wants to use them.